SayTheirNames/Source/Model/Image.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Images.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+public struct Image: Decodable {
+    var id: Int
+    var personId: Int
```

Comment:
by swift capitalisation rules it should be `personID`, `requestURL`, etc

Reply:
Fixed.